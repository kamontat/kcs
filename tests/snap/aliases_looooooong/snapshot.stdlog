00:10:45 [INF] private.base.settings : testing mode has been enabled
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_tmp_hook_setup' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_long_hook_main' function with 1 args [cmd.long]
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_environment_hook_clean' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_configs_hook_clean' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_tmp_hook_clean' function with 0 args []
00:10:45 [DBG] libs.functions.loader.run : run '__kcs_log_hook_clean' function with 0 args []
