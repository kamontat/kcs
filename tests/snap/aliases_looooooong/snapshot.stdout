00:10:45 [PRT] cmd.long : execute long command
00:10:45 [PRT] cmd.long : key      : long
00:10:45 [PRT] cmd.long : name     : long
00:10:45 [PRT] cmd.long : args     : 1 (cmd.long)
00:10:45 [PRT] cmd.long : cmd args : 0 ()
